docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreweaviate.md

```diff
@@ -0,0 +1,141 @@
+///
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/sub-node-expression-resolution.md"
+
+## Node usage patterns
+
+You can use the Weaviate Vector Store node in the following patterns.
+
+### Use as a regular node to insert and retrieve documents
+
+You can use the Weaviate Vector Store as a regular node to insert or get documents. This pattern places the Weaviate Vector Store in the regular connection flow without using an agent.
+
+
+### Connect directly to an AI agent as a tool
+
+You can connect the Weaviate Vector Store node directly to the tool connector of an [AI agent](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent/index.md) to use a vector store as a resource when answering queries.
+
+Here, the connection would be: AI agent (tools connector) -> Weaviate Vector Store node.
+
+### Use a retriever to fetch documents
+
+You can use the [Vector Store Retriever](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.retrievervectorstore.md) node with the Weaviate Vector Store node to fetch documents from the Weaviate Vector Store node. This is often used with the [Question and Answer Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainretrievalqa/index.md) node to fetch documents from the vector store that match the given chat input.
+
+
+### Use the Vector Store Question Answer Tool to answer questions
+
+Another pattern uses the [Vector Store Question Answer Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolvectorstore.md) to summarize results and answer questions from the Weaviate Vector Store node. Rather than connecting the Weaviate Vector Store directly as a tool, this pattern uses a tool specifically designed to summarizes data in the vector store.
+
+	
+## Node parameters
+
+/// note | Multi Tenancy
+You can separate your data into isolated tenants for the same collection (eg different customers). For that, you must always provide a `Tenant` both for inserting and retrieving objects. [Read more about multi tenancy in Weaviate docs.](https://weaviate.io/developers/weaviate/manage-data/multi-tenancy)
+///
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-mode.md"
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-rerank-results.md"
+
+
+<!-- vale from-write-good.Weasel = NO -->
+### Get Many parameters
+<!-- vale from-write-good.Weasel = YES -->
+
+* **Weaviate collection**: Enter the name of the Weaviate collection to use.
+* **Prompt**: Enter the search query.
+* **Limit**: Enter how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+
+This Operation Mode includes one **Node option**, the [Metadata Filter](#metadata-filter).
+
+### Insert Documents parameters
+
+* **Weaviate collection**: Enter the name of the Weaviate collection to use.
+
```

Comment:
Looks like there's also "Embedding Batch Size" missing here.

Reply:
This one seems to be a shared one a common to all nodes 🤔